src/js/settings.js

```diff
@@ -0,0 +1,61 @@
+import { setting } from './translation.js'
+
+const settings = document.querySelector('.settings');
+const settingsBtn = document.querySelector('.icon_settings');
+const elementsList = document.querySelectorAll('.block input');
+const elementsListText = document.querySelectorAll('.block label');
+const textCahngeLang = document.querySelector('.title__language');
+const textChangePhoto = document.querySelector('.title__photo');
+const ruLang = document.querySelector('option[value="ru"]');
+const enLang = document.querySelector('option[value="en"]');
+
+let isOpen = false;
+
+export const openCloseSettings = () => {
+  settingsBtn.addEventListener('click', () => {
+    if (isOpen === false) {
+      isOpen = true;
+      settings.style.transform = `translateX(0)`;
+    } else if (isOpen === true) {
+      isOpen = false;
+      settings.style.transform = `translateX(-120%)`;
+    }
```

Comment:
```js
settings.style.transform = isOpen ? `translateX(-120%)` : `translateX(0)`;
isOpen = !isOpen
```
2 lines of code do the same thing as your 6 lines of code. no code means no bugs